docs/src/writing-tests-js-python.md

```diff
@@ -52,11 +52,38 @@ test('get started link', async ({ page }) => {
 });
 ```
 
+```python title="test_example.py"
+import re
+from playwright.sync_api import Page, expect
+
+
+def test_homepage_has_Playwright_in_title_and_get_started_link_linking_to_the_intro_page(page: Page):
```

Comment:
```suggestion
def test_homepage_has_playwright_in_title_and_get_started_link_linking_to_the_intro_page(page: Page):
```

Reply:
should we split this into 2 tests like we do in node so its same experience or is there a reason why in python we have it as only one???

Reply:
two tests sounds good.